docs/azure_arc_jumpstart/azure_arc_k8s/day2/microk8s/local_microk8s_gitops_helm/_index.md

```diff
@@ -123,157 +117,198 @@ The demo application that will be deployed later in this scenario relies on an i
+    ![Screenshot showing Azure portal with no Azure Arc-enabled Kubernetes GitOps configurations](./08.png)
 
 ### Deployment Flow
 
-For our scenario, we will deploy the "Hello Arc" application from the ["demo repository"](https://github.com/likamrat/hello_arc) through GitOps. We will deploy the "Hello Arc" application (a Namespace-level component) with 1 replica to the *prod* namespace.
+For our scenario, notice we have Helm chart in the "Azure Arc Jumpstart Apps" repository for the Hello-Arc application as well as the Helm Release.
+
+!["Azure Arc Jumpstart Apps" GitHub repository](./09.png)
 
-!["Hello Arc" application GitHub repository](./08.png)
+!["Azure Arc Jumpstart Apps" GitHub repository](./10.png)
 
-!["Hello Arc" application GitHub repository](./09.png)
+- The "Hello Arc" application (a Namespace-level component) will be deployed with 3 replica to the _hello-arc_ namespace.
 
 ### Deployment
 
 To create the GitOps configuration and it's respective Kubernetes resources, we've provided a script in a shell file (Linux, MacOS) and batch file (Windows).
 
-* Linux and MacOS
+- Linux and MacOS
 
-    Edit the environment variables in the [*az_k8sconfig_helm_microk8s*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/microk8s/gitops/helm/az_k8sconfig_helm_microk8s.sh) shell script to match your parameters, and run the below command.
+    Edit the environment variables in the [_az_k8sconfig_helm_microk8s_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/microk8s/gitops/helm/az_k8sconfig_helm_microk8s.sh) shell script to match your parameters, and run the below command.
 
     ```shell
     . ./az_k8sconfig_helm_microk8s.sh
     ```
 
-    > **NOTE: The extra dot is due to the script having an *export* function and that needs to have the vars exported in the same shell session as the rest of the commands.**
+    > **NOTE: The extra dot is due to the script having an _export_ function and that needs to have the vars exported in the same shell session as the rest of the commands.**
 
-* Windows
+- Windows
 
-    Edit the environment variables in the [*az_k8sconfig_helm_microk8s_windows*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/microk8s/gitops/helm/az_k8sconfig_helm_microk8s_windows.ps1) PowerShell file to match your parameters, and run it using the ```.\az_k8sconfig_helm_microk8s.ps1``` command.
+    Edit the environment variables in the [_az_k8sconfig_helm_microk8s_windows_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/microk8s/gitops/helm/az_k8sconfig_helm_microk8s_windows.ps1) PowerShell file to match your parameters, and run it using the ```.\az_k8sconfig_helm_microk8s.ps1``` command.
 
-The `az_k8sconfig_helm_microk8s` and `az_k8sconfig_helm_microk8s_windows` scripts will:
+    For example:
+
+    ![Screenshot parameter examples](./11.png)
```

Comment:
I know it's an example, make sure to either mask the creds in the screenshot or delete the SP